test/integration/usersFilter.test.js

```diff
@@ -48,6 +49,8 @@ describe("Filter Users", function () {
     await updateUserStatus(idleUser, generateUserStatusData("IDLE", updatedAtDate, updatedAtDate, untilDate, "CSS"));
     activeUser = await addUser(userData[8]);
     await updateUserStatus(activeUser, generateUserStatusData("ACTIVE", updatedAtDate, updatedAtDate));
+    onboardingUser = await addUser(userData[2]);
+    await updateUserStatus(onboardingUser, generateUserStatusData("ONBOARDING", updatedAtDate, updatedAtDate));
```

Comment:
can we use `ONBOARDING ` here and in the previous line from userState constant variable

Reply:
Sorry, but i am using ONBOARDING at line 53?

Reply:
yes if see there is a constant called `userState.ONBOARDING` in `constants/userStatus.js` you can use that. Also in the previous lines where ACTIVE , IDLE , OOO is used. import from there and use